src/databricks/labs/ucx/workspace_access/generic.py

```diff
@@ -110,21 +109,19 @@ def _response_to_request(
 
     @rate_limited(max_requests=30)
     def _applier_task(self, object_type: str, object_id: str, acl: list[iam.AccessControlRequest]):
```

Comment:
If you remove the loop here, you must add "@retried(on=[this], timeout=timedelta(minutes=1))", preferably you make time delta configurable in the unit tests, otherwise they'll run slower. 


At this moment this change is a regression and introduces more bugs than it fixes

Reply:
Leave me alone

Reply:
Does it make sense to add `@retried` on top if it's already present in the 2 function calls?

Reply:
@tnguyen-db see https://github.com/databrickslabs/ucx/blob/bc7f29e0727c7b38d53f7c43e194fbe653bf46d6/src/databricks/labs/ucx/workspace_access/secrets.py#L92C1-L97